### ISSUE TITLE
Quorum reelection loop under heavy load

### ISSUE THREAD

**Issue by ThomasRalee** [Consensys-Incorporated/quorum#825]
Hi,
i am not sure if what i've encountered is related to this issue logged on [etcd -Leadership changes under heavy load](https://github.com/etcd-io/etcd/issues/8553). It seems pretty similar, these are more research done on [ETCD - heartbeat interval config](https://github.com/jpmorganchase/quorum/blob/58fc5cd59d68c4c3e658f995b179713c7cc47336/raft/handler.go#L440)

Logging this issue here hopefully to get some additional insights. 

#### System information

Geth version: `2.2.5` - RAFT
Tessera Version: `0.10.0`

#### Expected behaviour
Election should take plan smoothly

#### Actual behaviour
Reelection happens on 2 seconds interval

#### Steps to reproduce the behaviour
- this is a very rare issue that happened, I was unable to reproduce this on a step-by-step basis 

#### Backtrace - filtered logs to show how often raft leadership election was taking place.
````
"raft2019/09/19 08:18:07 INFO: 1 became candidate at term 287
"
"raft2019/09/19 08:18:07 INFO: 1 is starting a new election at term 286
"
"raft2019/09/19 08:18:05 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 286
"
"raft2019/09/19 08:18:05 INFO: 1 received MsgVoteResp from 1 at term 286
"
"raft2019/09/19 08:18:05 INFO: 1 became candidate at term 286
"
"raft2019/09/19 08:18:05 INFO: 1 is starting a new election at term 285
"
"raft2019/09/19 08:18:04 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 285
"
"raft2019/09/19 08:18:04 INFO: 1 received MsgVoteResp from 1 at term 285
"
"raft2019/09/19 08:18:04 INFO: 1 became candidate at term 285
"
"raft2019/09/19 08:18:04 INFO: 1 is starting a new election at term 284
"
"raft2019/09/19 08:18:02 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 284
"
"raft2019/09/19 08:18:02 INFO: 1 received MsgVoteResp from 1 at term 284
"
"raft2019/09/19 08:18:02 INFO: 1 became candidate at term 284
"
"raft2019/09/19 08:18:02 INFO: 1 is starting a new election at term 283
"
"raft2019/09/19 08:18:01 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 283
"
"raft2019/09/19 08:18:01 INFO: 1 received MsgVoteResp from 1 at term 283
"
"raft2019/09/19 08:18:01 INFO: 1 became candidate at term 283
"
"raft2019/09/19 08:18:01 INFO: 1 is starting a new election at term 282
"
"raft2019/09/19 08:17:59 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 282
"
"raft2019/09/19 08:17:59 INFO: 1 received MsgVoteResp from 1 at term 282
"
"raft2019/09/19 08:17:59 INFO: 1 became candidate at term 282
"
"raft2019/09/19 08:17:59 INFO: 1 is starting a new election at term 281
"
"raft2019/09/19 08:17:58 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 281
"
"raft2019/09/19 08:17:58 INFO: 1 received MsgVoteResp from 1 at term 281
"
"raft2019/09/19 08:17:58 INFO: 1 became candidate at term 281
"
"raft2019/09/19 08:17:58 INFO: 1 is starting a new election at term 280
"
"raft2019/09/19 08:17:56 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 280
"
"raft2019/09/19 08:17:56 INFO: 1 received MsgVoteResp from 1 at term 280
"
"raft2019/09/19 08:17:56 INFO: 1 became candidate at term 280
"
"raft2019/09/19 08:17:56 INFO: 1 is starting a new election at term 279
"
"raft2019/09/19 08:17:55 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 279
"
"raft2019/09/19 08:17:55 INFO: 1 received MsgVoteResp from 1 at term 279
"
"raft2019/09/19 08:17:55 INFO: 1 became candidate at term 279
"
"raft2019/09/19 08:17:55 INFO: 1 is starting a new election at term 278
"
"raft2019/09/19 08:17:53 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 278
"
"raft2019/09/19 08:17:53 INFO: 1 received MsgVoteResp from 1 at term 278
"
"raft2019/09/19 08:17:53 INFO: 1 became candidate at term 278
"
"raft2019/09/19 08:17:53 INFO: 1 is starting a new election at term 277
"
"raft2019/09/19 08:17:52 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 277
"
"raft2019/09/19 08:17:52 INFO: 1 received MsgVoteResp from 1 at term 277
"
"raft2019/09/19 08:17:52 INFO: 1 became candidate at term 277
"
"raft2019/09/19 08:17:52 INFO: 1 is starting a new election at term 276
"
"raft2019/09/19 08:17:51 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 276
"
"raft2019/09/19 08:17:51 INFO: 1 received MsgVoteResp from 1 at term 276
"
"raft2019/09/19 08:17:51 INFO: 1 became candidate at term 276
"
"raft2019/09/19 08:17:51 INFO: 1 is starting a new election at term 275
"
"raft2019/09/19 08:17:49 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 275
"
"raft2019/09/19 08:17:49 INFO: 1 received MsgVoteResp from 1 at term 275
"
"raft2019/09/19 08:17:49 INFO: 1 became candidate at term 275
"
"raft2019/09/19 08:17:49 INFO: 1 is starting a new election at term 274
"
"raft2019/09/19 08:17:47 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 274
"
"raft2019/09/19 08:17:47 INFO: 1 received MsgVoteResp from 1 at term 274
"
"raft2019/09/19 08:17:47 INFO: 1 became candidate at term 274
"
"raft2019/09/19 08:17:47 INFO: 1 is starting a new election at term 273
"
"raft2019/09/19 08:17:46 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 273
"
"raft2019/09/19 08:17:46 INFO: 1 received MsgVoteResp from 1 at term 273
"
"raft2019/09/19 08:17:46 INFO: 1 became candidate at term 273
"
"raft2019/09/19 08:17:46 INFO: 1 is starting a new election at term 272
"
"raft2019/09/19 08:17:44 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 272
"
"raft2019/09/19 08:17:44 INFO: 1 received MsgVoteResp from 1 at term 272
"
"raft2019/09/19 08:17:44 INFO: 1 became candidate at term 272
"
"raft2019/09/19 08:17:44 INFO: 1 is starting a new election at term 271
"
"raft2019/09/19 08:17:42 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 271
"
"raft2019/09/19 08:17:42 INFO: 1 received MsgVoteResp from 1 at term 271
"
"raft2019/09/19 08:17:42 INFO: 1 became candidate at term 271
"
"raft2019/09/19 08:17:42 INFO: 1 is starting a new election at term 270
"
"raft2019/09/19 08:17:40 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 270
"
"raft2019/09/19 08:17:40 INFO: 1 received MsgVoteResp from 1 at term 270
"
"raft2019/09/19 08:17:40 INFO: 1 became candidate at term 270
"
"raft2019/09/19 08:17:40 INFO: 1 is starting a new election at term 269
"
"raft2019/09/19 08:17:39 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 269
"
"raft2019/09/19 08:17:39 INFO: 1 received MsgVoteResp from 1 at term 269
"
"raft2019/09/19 08:17:39 INFO: 1 became candidate at term 269
"
"raft2019/09/19 08:17:39 INFO: 1 is starting a new election at term 268
"
"raft2019/09/19 08:17:37 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 268
"
"raft2019/09/19 08:17:37 INFO: 1 received MsgVoteResp from 1 at term 268
"
"raft2019/09/19 08:17:37 INFO: 1 became candidate at term 268
"
"raft2019/09/19 08:17:37 INFO: 1 is starting a new election at term 267
"
"2019-09-19 08:17:37.099 [INFO][9] startup.go 280: Determined node name: gke-dev-asia-network-n1-highmem-2-pre-af13929f-mrt0
"
"raft2019/09/19 08:17:35 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 267
"
"raft2019/09/19 08:17:35 INFO: 1 received MsgVoteResp from 1 at term 267
"
"raft2019/09/19 08:17:35 INFO: 1 became candidate at term 267
"
"raft2019/09/19 08:17:35 INFO: 1 is starting a new election at term 266
"
"raft2019/09/19 08:17:34 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 266
"
"raft2019/09/19 08:17:34 INFO: 1 received MsgVoteResp from 1 at term 266
"
"raft2019/09/19 08:17:34 INFO: 1 became candidate at term 266
"
"raft2019/09/19 08:17:34 INFO: 1 is starting a new election at term 265
"
"raft2019/09/19 08:17:33 INFO: 2 [commit: 3] restored snapshot [index: 3, term: 2]
"
"raft2019/09/19 08:17:33 INFO: log [committed=0, applied=0, unstable.offset=1, len(unstable.Entries)=0] starts to restore snapshot [index: 3, term: 2]
"
"raft2019/09/19 08:17:33 INFO: 2 [commit: 0, lastindex: 0, lastterm: 0] starts to restore snapshot [index: 3, term: 2]
"
"raft2019/09/19 08:17:33 INFO: raft.node: 2 elected leader 1 at term 2
"
"raft2019/09/19 08:17:33 INFO: 2 became follower at term 2
"
"raft2019/09/19 08:17:33 INFO: 2 [term: 1] received a MsgHeartbeat message with higher term from 1 [term: 2]
"
"raft2019/09/19 08:17:33 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 265
"
"raft2019/09/19 08:17:33 INFO: 1 received MsgVoteResp from 1 at term 265
"
"raft2019/09/19 08:17:33 INFO: 1 became candidate at term 265
"
"raft2019/09/19 08:17:33 INFO: 1 is starting a new election at term 264
"
"raft2019/09/19 08:17:33 INFO: 2 became follower at term 1
"
"raft2019/09/19 08:17:33 INFO: newRaft 2 [peers: [], term: 0, commit: 0, applied: 0, lastindex: 0, lastterm: 0]
"
"raft2019/09/19 08:17:33 INFO: 2 became follower at term 0
"
"INFO [09-19|08:17:33.189] loading WAL                              term=0 index=0
"
"raft2019/09/19 08:17:31 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 264
"
"raft2019/09/19 08:17:31 INFO: 1 received MsgVoteResp from 1 at term 264
"
"raft2019/09/19 08:17:31 INFO: 1 became candidate at term 264
"
"raft2019/09/19 08:17:31 INFO: 1 is starting a new election at term 263
"
"raft2019/09/19 08:17:30 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 263
"
"raft2019/09/19 08:17:30 INFO: 1 received MsgVoteResp from 1 at term 263
"
"raft2019/09/19 08:17:30 INFO: 1 became candidate at term 263
"
"raft2019/09/19 08:17:30 INFO: 1 is starting a new election at term 262
"
"raft2019/09/19 08:17:29 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 262
"
"raft2019/09/19 08:17:29 INFO: 1 received MsgVoteResp from 1 at term 262
"
"raft2019/09/19 08:17:29 INFO: 1 became candidate at term 262
"
"raft2019/09/19 08:17:29 INFO: 1 is starting a new election at term 261
"
"raft2019/09/19 08:17:28 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 261
"
"raft2019/09/19 08:17:28 INFO: 1 received MsgVoteResp from 1 at term 261
"
"raft2019/09/19 08:17:28 INFO: 1 became candidate at term 261
"
"raft2019/09/19 08:17:28 INFO: 1 is starting a new election at term 260
"
"raft2019/09/19 08:17:27 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 260
"
"raft2019/09/19 08:17:27 INFO: 1 received MsgVoteResp from 1 at term 260
"
"raft2019/09/19 08:17:27 INFO: 1 became candidate at term 260
"
"raft2019/09/19 08:17:27 INFO: 1 is starting a new election at term 259
"
"raft2019/09/19 08:17:26 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 259
"
"raft2019/09/19 08:17:26 INFO: 1 received MsgVoteResp from 1 at term 259
"
"raft2019/09/19 08:17:26 INFO: 1 became candidate at term 259
"
"raft2019/09/19 08:17:26 INFO: 1 is starting a new election at term 258
"
"raft2019/09/19 08:17:24 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 258
"
"raft2019/09/19 08:17:24 INFO: 1 received MsgVoteResp from 1 at term 258
"
"raft2019/09/19 08:17:24 INFO: 1 became candidate at term 258
"
"raft2019/09/19 08:17:24 INFO: 1 is starting a new election at term 257
"
"raft2019/09/19 08:17:23 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 257
"
"raft2019/09/19 08:17:23 INFO: 1 received MsgVoteResp from 1 at term 257
"
"raft2019/09/19 08:17:23 INFO: 1 became candidate at term 257
"
"raft2019/09/19 08:17:23 INFO: 1 is starting a new election at term 256
"
"raft2019/09/19 08:17:21 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 256
"
"raft2019/09/19 08:17:21 INFO: 1 received MsgVoteResp from 1 at term 256
"
"raft2019/09/19 08:17:21 INFO: 1 became candidate at term 256
"
"raft2019/09/19 08:17:21 INFO: 1 is starting a new election at term 255
"
"raft2019/09/19 08:17:20 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 255
"
"raft2019/09/19 08:17:20 INFO: 1 received MsgVoteResp from 1 at term 255
"
"raft2019/09/19 08:17:20 INFO: 1 became candidate at term 255
"
"raft2019/09/19 08:17:20 INFO: 1 is starting a new election at term 254
"
"raft2019/09/19 08:17:19 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 254
"
"raft2019/09/19 08:17:19 INFO: 1 received MsgVoteResp from 1 at term 254
"
"raft2019/09/19 08:17:19 INFO: 1 became candidate at term 254
"
"raft2019/09/19 08:17:19 INFO: 1 is starting a new election at term 253
"
"raft2019/09/19 08:17:18 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 253
"
"raft2019/09/19 08:17:18 INFO: 1 received MsgVoteResp from 1 at term 253
"
"raft2019/09/19 08:17:18 INFO: 1 became candidate at term 253
"
"raft2019/09/19 08:17:18 INFO: 1 is starting a new election at term 252
"
"raft2019/09/19 08:17:16 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 252
"
"raft2019/09/19 08:17:16 INFO: 1 received MsgVoteResp from 1 at term 252
"
"raft2019/09/19 08:17:16 INFO: 1 became candidate at term 252
"
"raft2019/09/19 08:17:16 INFO: 1 is starting a new election at term 251
"
"raft2019/09/19 08:17:15 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 251
"
"raft2019/09/19 08:17:15 INFO: 1 received MsgVoteResp from 1 at term 251
"
"raft2019/09/19 08:17:15 INFO: 1 became candidate at term 251
"
"raft2019/09/19 08:17:15 INFO: 1 is starting a new election at term 250
"
"raft2019/09/19 08:17:13 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 250
"
"raft2019/09/19 08:17:13 INFO: 1 received MsgVoteResp from 1 at term 250
"
"raft2019/09/19 08:17:13 INFO: 1 became candidate at term 250
"
"raft2019/09/19 08:17:13 INFO: 1 is starting a new election at term 249
"
"raft2019/09/19 08:17:12 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 249
"
"raft2019/09/19 08:17:12 INFO: 1 received MsgVoteResp from 1 at term 249
"
"raft2019/09/19 08:17:12 INFO: 1 became candidate at term 249
"
"raft2019/09/19 08:17:12 INFO: 1 is starting a new election at term 248
"
"raft2019/09/19 08:17:11 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 248
"
"raft2019/09/19 08:17:11 INFO: 1 received MsgVoteResp from 1 at term 248
"
"raft2019/09/19 08:17:11 INFO: 1 became candidate at term 248
"
"raft2019/09/19 08:17:11 INFO: 1 is starting a new election at term 247
"
"raft2019/09/19 08:17:10 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 247
"
"raft2019/09/19 08:17:10 INFO: 1 received MsgVoteResp from 1 at term 247
"
"raft2019/09/19 08:17:10 INFO: 1 became candidate at term 247
"
"raft2019/09/19 08:17:10 INFO: 1 is starting a new election at term 246
"
"raft2019/09/19 08:17:08 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 246
"
"raft2019/09/19 08:17:08 INFO: 1 received MsgVoteResp from 1 at term 246
"
"raft2019/09/19 08:17:08 INFO: 1 became candidate at term 246
"
"raft2019/09/19 08:17:08 INFO: 1 is starting a new election at term 245
"
"raft2019/09/19 08:17:07 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 245
"
"raft2019/09/19 08:17:07 INFO: 1 received MsgVoteResp from 1 at term 245
"
"raft2019/09/19 08:17:07 INFO: 1 became candidate at term 245
"
"raft2019/09/19 08:17:07 INFO: 1 is starting a new election at term 244
"
"raft2019/09/19 08:17:06 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 244
"
"raft2019/09/19 08:17:06 INFO: 1 received MsgVoteResp from 1 at term 244
"
"raft2019/09/19 08:17:06 INFO: 1 became candidate at term 244
"
"raft2019/09/19 08:17:06 INFO: 1 is starting a new election at term 243
"
"raft2019/09/19 08:17:04 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 243
"
"raft2019/09/19 08:17:04 INFO: 1 received MsgVoteResp from 1 at term 243
"
"raft2019/09/19 08:17:04 INFO: 1 became candidate at term 243
"
"raft2019/09/19 08:17:04 INFO: 1 is starting a new election at term 242
"
"raft2019/09/19 08:17:03 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 242
"
"raft2019/09/19 08:17:03 INFO: 1 received MsgVoteResp from 1 at term 242
"
"raft2019/09/19 08:17:03 INFO: 1 became candidate at term 242
"
"raft2019/09/19 08:17:03 INFO: 1 is starting a new election at term 241
"
"raft2019/09/19 08:17:02 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 241
"
"raft2019/09/19 08:17:02 INFO: 1 received MsgVoteResp from 1 at term 241
"
"raft2019/09/19 08:17:02 INFO: 1 became candidate at term 241
"
"raft2019/09/19 08:17:02 INFO: 1 is starting a new election at term 240
"
"raft2019/09/19 08:17:00 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 240
"
"raft2019/09/19 08:17:00 INFO: 1 received MsgVoteResp from 1 at term 240
"
"raft2019/09/19 08:17:00 INFO: 1 became candidate at term 240
"
"raft2019/09/19 08:17:00 INFO: 1 is starting a new election at term 239
"
"raft2019/09/19 08:16:59 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 239
"
"raft2019/09/19 08:16:59 INFO: 1 received MsgVoteResp from 1 at term 239
"
"raft2019/09/19 08:16:59 INFO: 1 became candidate at term 239
"
"raft2019/09/19 08:16:59 INFO: 1 is starting a new election at term 238
"
"raft2019/09/19 08:16:57 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 238
"
"raft2019/09/19 08:16:57 INFO: 1 received MsgVoteResp from 1 at term 238
"
"raft2019/09/19 08:16:57 INFO: 1 became candidate at term 238
"
"raft2019/09/19 08:16:57 INFO: 1 is starting a new election at term 237
"
"raft2019/09/19 08:16:56 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 237
"
"raft2019/09/19 08:16:56 INFO: 1 received MsgVoteResp from 1 at term 237
"
"raft2019/09/19 08:16:56 INFO: 1 became candidate at term 237
"
"raft2019/09/19 08:16:56 INFO: 1 is starting a new election at term 236
"
"raft2019/09/19 08:16:54 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 236
"
"raft2019/09/19 08:16:54 INFO: 1 received MsgVoteResp from 1 at term 236
"
"raft2019/09/19 08:16:54 INFO: 1 became candidate at term 236
"
"raft2019/09/19 08:16:54 INFO: 1 is starting a new election at term 235
"
"raft2019/09/19 08:16:53 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 235
"
"raft2019/09/19 08:16:53 INFO: 1 received MsgVoteResp from 1 at term 235
"
"raft2019/09/19 08:16:53 INFO: 1 became candidate at term 235
"
"raft2019/09/19 08:16:53 INFO: 1 is starting a new election at term 234
"
"raft2019/09/19 08:16:52 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 234
"
"raft2019/09/19 08:16:52 INFO: 1 received MsgVoteResp from 1 at term 234
"
"raft2019/09/19 08:16:52 INFO: 1 became candidate at term 234
"
"raft2019/09/19 08:16:52 INFO: 1 is starting a new election at term 233
"
"raft2019/09/19 08:16:50 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 233
"
"raft2019/09/19 08:16:50 INFO: 1 received MsgVoteResp from 1 at term 233
"
"raft2019/09/19 08:16:50 INFO: 1 became candidate at term 233
"
"raft2019/09/19 08:16:50 INFO: 1 is starting a new election at term 232
"
"raft2019/09/19 08:16:49 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 232
"
"raft2019/09/19 08:16:49 INFO: 1 received MsgVoteResp from 1 at term 232
"
"raft2019/09/19 08:16:49 INFO: 1 became candidate at term 232
"
"raft2019/09/19 08:16:49 INFO: 1 is starting a new election at term 231
"
"raft2019/09/19 08:16:47 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 231
"
"raft2019/09/19 08:16:47 INFO: 1 received MsgVoteResp from 1 at term 231
"
"raft2019/09/19 08:16:47 INFO: 1 became candidate at term 231
"
"raft2019/09/19 08:16:47 INFO: 1 is starting a new election at term 230
"
"raft2019/09/19 08:16:46 INFO: raft.node: 2 elected leader 3 at term 8
"
"raft2019/09/19 08:16:46 INFO: newRaft 2 [peers: [1,2,3], term: 8, commit: 149, applied: 149, lastindex: 149, lastterm: 8]
"
"raft2019/09/19 08:16:46 INFO: 2 became follower at term 8
"
"INFO [09-19|08:16:46.807] loading WAL                              term=0 index=0
"
"raft2019/09/19 08:16:46 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 230
"
"raft2019/09/19 08:16:46 INFO: 1 received MsgVoteResp from 1 at term 230
"
"raft2019/09/19 08:16:46 INFO: 1 became candidate at term 230
"
"raft2019/09/19 08:16:46 INFO: 1 is starting a new election at term 229
"
"raft2019/09/19 08:16:45 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 229
"
"raft2019/09/19 08:16:45 INFO: 1 received MsgVoteResp from 1 at term 229
"
"raft2019/09/19 08:16:45 INFO: 1 became candidate at term 229
"
"raft2019/09/19 08:16:45 INFO: 1 is starting a new election at term 228
"
"raft2019/09/19 08:16:43 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 228
"
"raft2019/09/19 08:16:43 INFO: 1 received MsgVoteResp from 1 at term 228
"
"raft2019/09/19 08:16:43 INFO: 1 became candidate at term 228
"
"raft2019/09/19 08:16:43 INFO: 1 is starting a new election at term 227
"
"raft2019/09/19 08:16:42 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 227
"
"raft2019/09/19 08:16:42 INFO: 1 received MsgVoteResp from 1 at term 227
"
"raft2019/09/19 08:16:42 INFO: 1 became candidate at term 227
"
"raft2019/09/19 08:16:42 INFO: 1 is starting a new election at term 226
"
"raft2019/09/19 08:16:42 INFO: newRaft 1 [peers: [], term: 2, commit: 2, applied: 2, lastindex: 2, lastterm: 2]
"
"raft2019/09/19 08:16:42 INFO: 1 became follower at term 2
"
"INFO [09-19|08:16:42.467] loading WAL                              term=0 index=0
"
"raft2019/09/19 08:16:41 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 226
"
"raft2019/09/19 08:16:41 INFO: 1 received MsgVoteResp from 1 at term 226
"
"raft2019/09/19 08:16:41 INFO: 1 became candidate at term 226
"
"raft2019/09/19 08:16:41 INFO: 1 is starting a new election at term 225
"
"raft2019/09/19 08:16:40 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 225
"
"raft2019/09/19 08:16:40 INFO: 1 received MsgVoteResp from 1 at term 225
"
"raft2019/09/19 08:16:40 INFO: 1 became candidate at term 225
"
"raft2019/09/19 08:16:40 INFO: 1 is starting a new election at term 224
"
"raft2019/09/19 08:16:38 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 224
"
"raft2019/09/19 08:16:38 INFO: 1 received MsgVoteResp from 1 at term 224
"
"raft2019/09/19 08:16:38 INFO: 1 became candidate at term 224
"
"raft2019/09/19 08:16:38 INFO: 1 is starting a new election at term 223
"
"raft2019/09/19 08:16:37 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 223
"
"raft2019/09/19 08:16:37 INFO: 1 received MsgVoteResp from 1 at term 223
"
"raft2019/09/19 08:16:37 INFO: 1 became candidate at term 223
"
"raft2019/09/19 08:16:37 INFO: 1 is starting a new election at term 222
"
"raft2019/09/19 08:16:36 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 222
"
"raft2019/09/19 08:16:36 INFO: 1 received MsgVoteResp from 1 at term 222
"
"raft2019/09/19 08:16:36 INFO: 1 became candidate at term 222
"
"raft2019/09/19 08:16:36 INFO: 1 is starting a new election at term 221
"
"raft2019/09/19 08:16:35 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 221
"
"raft2019/09/19 08:16:35 INFO: 1 received MsgVoteResp from 1 at term 221
"
"raft2019/09/19 08:16:35 INFO: 1 became candidate at term 221
"
"raft2019/09/19 08:16:35 INFO: 1 is starting a new election at term 220
"
"raft2019/09/19 08:16:33 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 220
"
"raft2019/09/19 08:16:33 INFO: 1 received MsgVoteResp from 1 at term 220
"
"raft2019/09/19 08:16:33 INFO: 1 became candidate at term 220
"
"raft2019/09/19 08:16:33 INFO: 1 is starting a new election at term 219
"
"raft2019/09/19 08:16:32 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 219
"
"raft2019/09/19 08:16:32 INFO: 1 received MsgVoteResp from 1 at term 219
"
"raft2019/09/19 08:16:32 INFO: 1 became candidate at term 219
"
"raft2019/09/19 08:16:32 INFO: 1 is starting a new election at term 218
"
"raft2019/09/19 08:16:30 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 218
"
"raft2019/09/19 08:16:30 INFO: 1 received MsgVoteResp from 1 at term 218
"
"raft2019/09/19 08:16:30 INFO: 1 became candidate at term 218
"
"raft2019/09/19 08:16:30 INFO: 1 is starting a new election at term 217
"
"raft2019/09/19 08:16:29 INFO: 1 [logterm: 2, index: 4] sent MsgVote request to 2 at term 217
"
"raft2019/09/19 08:16:29 INFO: 1 received MsgVoteResp from 1 at term 217
"
"raft2019/09/19 08:16:29 INFO: 1 became candidate at term 217
"
"raft2019/09/19 08:16:29 INFO: 1 is starting a new election at term 216
"
````

**Comment by vsmk98** [comment]
@ThomasRalee thanks for your patience on this issue. We are looking into the suggested changes and testing the same. Will revert soon once we complete the analysis.

**Comment by fixanoid** [comment]
Should be addressed by 2.4.0 release.
